tests/execute/result/repeated.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "run=\$(mktemp -d)" 0 "Create run directory"
+        rlRun "pushd repeated"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Running tests separately"
+        rlRun -s "tmt run -v -i $run" 1
+
+        # Check output
+        rlAssertGrep "pass /test" "$rlRun_LOG"
```

Comment:
This variable is always set by `beakerlib`. Can we ignore specific variables?

Reply:
I can ask some friends, if there is some way to do this, looking @ https://github.com/koalaman/shellcheck/wiki/Ignore

does not feel like it :(

Reply:
Asked the friends, seems it is not easy now, here are some relevant upstream issues:

https://github.com/koalaman/shellcheck/issues/356
https://github.com/koalaman/shellcheck/issues/2356

Reply:
Setting the default value in case of `rlRun_LOG` being unset should make ShellCheck happy.

```suggestion
        rlAssertGrep "pass /test" "${rlRun_LOG:-}"
```

Reply:
Hm, I'm not sure we want to add such "noise" into the shell/test code. I'd rather go with defining set of known variables as suggested in koalaman/shellcheck/issues/2356.